modules/Microsoft.DevTestLab/labs/deploy.bicep

```diff
@@ -61,6 +64,36 @@ param userAssignedIdentities object = {}
 @description('Optional. The ID(s) to assign to the virtual machines associated with this lab.')
 param managementIdentities object = {}
 
+@description('Optional. Resource Group allocation for virtual machines. If left empty, virtual machines will be deployed in their own Resource Groups. Default is the same Resource Group for DevTest Lab.')
+param vmCreationResourceGroupId string = resourceGroup().id
+
+@allowed([
+  'Enabled'
+  'Disabled'
+])
+@description('Optional. Enable browser connect on virtual machines if the lab\'s VNETs have configured Azure Bastion. Default is "Disabled".')
+param browserConnect string = 'Disabled'
+
+@description('Optional. Disable auto upgrade custom script extension minor version. Default is false.')
+param disableAutoUpgradeCseMinorVersion bool = false
+
+@allowed([
+  'Enabled'
+  'Disabled'
+])
+@description('Optional. Enable lab resources isolation from the public internet. Default is "Enabled".')
+param isolateLabResources string = 'Enabled'
+
+@allowed([
+  'EncryptionAtRestWithPlatformKey'
+  'EncryptionAtRestWithCustomerKey'
+])
+@description('Optional. Specify how OS and data disks created as part of the lab are encrypted. Default is "EncryptionAtRestWithPlatformKey".')
```

Comment:
```suggestion
@description('Optional. Specify how OS and data disks created as part of the lab are encrypted.')
```

Reply:
Also, is there an 'allowed()' set?

Reply:
Yep. Right above it. These are the allowed string types.